frontend/src/views/CellGuide/components/TissueCard/index.tsx

```diff
@@ -36,6 +48,14 @@ export default function TissueCard(): JSX.Element {
     return () => window.removeEventListener("resize", onResize);
   }, []);
 
+  const [descriptionUberon, setDescriptionUberon] = useState<string>("");
```

Comment:
Feels like we might not need this state and can just use rawDescriptionUberon directly?

Reply:
@atarashansky PTAL thanks!

Reply:
Actually never mind, for SEO I'm going to fetch the description at build time, so I'll just remove this completely! Thanks!